Readme.md

```diff
@@ -68,6 +68,18 @@ composer test-all      # psalm, compiler & core tests after each other
 > composer test-core
 ```
 
+### Architecture Decision Record (ADR) 
+
+These helps us to bring more structure to our code. However, the most important thing with ADRs or RFCs is the process
```

Comment:
It should start with an Introduction to what "Substantial changes" are and what they are not. Substantial changes are architecture decisions, documentation restructuring, breaking changes, etc. But not Bug Reports, Bug Fixes, Unit Tests, etc.